masked-number/functions/relay-sms.protected.js

```diff
@@ -0,0 +1,33 @@
+exports.handler = async function(context, event, callback) {
+  const client = context.getTwilioClient();
+  const twiml = new Twilio.twiml.MessagingResponse();
+
+  if (event.From === context.MY_PHONE_NUMBER) {
+    const separatorPosition = event.Body.indexOf(':');
+
+    if (separatorPosition < 1) {
+      twiml.message('You need to specify a recipient number and a ":" before the message.');
```

Comment:
it might be helpful to provide a more specific example of how this should look in the response here